app.js

```diff
@@ -13,13 +14,45 @@ app.set('views', path.join(__dirname, 'views'));
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
-
+hbs.registerPartials(__dirname + '/views/partials')
 // ...
 
 // Add the route handlers here:
 
-app.get('/', (req, res) => {
-  res.render('index');
+app.get('/', (req, res) => res.render('index'));
+
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
```

Comment:
I would consider renaming "beersFromApi" to "beerFromApi" as this will be just one.